docs/kustomization.yaml

```diff
@@ -184,3 +184,20 @@ vars:
       apiVersion: apps/v1
     fieldref:
       fieldpath: spec.template.spec.restartPolicy
+
+# ImageTags are used to modify the tags for images used in different resources without creating patches.
+# For example, in deployment.yaml you have following container
+# ```
+#  containers:
+#    - name: myapp
+#      image: mycontainerregistry/myimage:v0
+#    - name: nginxapp
+#      image: nginx:1.7.9
+#```
+# You want to change the tag of myimage to v1 and change the tag of nginx to 1.8.0. You can add following
+# imageTags in kusotmization.yaml. Then kustomize build will replace the old tags with the new tags.
```

Comment:
one can change the tag of myimage to v1 and the tag of nginx to 1.8.0 with
the following:

Reply:
done